cmd/kubeadm/app/apis/kubeadm/v1alpha1/defaults.go

```diff
@@ -57,6 +61,10 @@ const (
 	DefaultEtcdCertDir = "/etc/kubernetes/pki/etcd"
 	// DefaultEtcdClusterServiceName is the default name of the service backing the etcd cluster
 	DefaultEtcdClusterServiceName = "etcd-cluster"
+	// DefaultProxyBindAddressv4 is the default bind address when the advertise address is v4
+	DefaultProxyBindAddressv4 = "0.0.0.0"
+	// DefaultProxyBindAddressv6 is the default bind address when the advertise address is v4
+	DefaultProxyBindAddressv6 = "\"::\""
```

Comment:
I think this should be `"::"` now that we're dealing with the config API. I understand the extra "" were needed for the flag, but don't think that's the case anymore. For instance, I couldn't get the proxy to load a file with the extra "", nor did that pass validation